community/approved-plugins.json

```diff
@@ -25,8 +25,23 @@
     "url": "https://github.com/nxkit/nxkit"
   },
   {
-    "name": "nx-plugins",
-    "description": "Nx plugin integrations with ESBuild / Vite / Snowpack / Prisma, with derived ESBuild / Snowpack / ... plugins.",
+    "name": "nx-plugin-esbuild",
+    "description": "Nx plugin integrations with ESBuild.",
+    "url": "https://nx-plugins.netlify.app/"
```

Comment:
Is this a closed source? How come we have no `github` url here?

Reply:
That is just the url that the plugin authors want people to go to when they click on the plugin details.  I get the repository url from the npm information.

Reply:
That particular plugin does have a github repo